app/assets/javascripts/discourse/app/components/create-account.js

```diff
@@ -52,22 +52,20 @@ export default Component.extend({
       }
     });
 
-    $(this.element).on("click.dropdown-user-field-label", "[for]", (event) => {
-      const $element = $(event.target);
-      const $target = $(`#${$element.attr("for")}`);
-
-      if ($target.is(".select-kit")) {
+    this.element.addEventListener("click", (event) => {
+      const target = document.getElementById(event.target.getAttribute("for"));
```

Comment:
Won't this error out when you click on something within the element that does not have the `for` attribute?

Reply:
Update: ah, it won't `document.getElementById()` is quite forgiving 😃